raiden/settings.py

```diff
@@ -46,6 +46,6 @@
 ETHERSCAN_API = 'https://{network}.etherscan.io/api?module=proxy&action={action}'
 
 RED_EYES_CONTRACT_VERSION = '0.4.0'
-DEVELOPMENT_CONTRACT_VERSION = '0.7.0'
+DEVELOPMENT_CONTRACT_VERSION = '0.6.0'
```

Comment:
The only valid versions are `0.3._`, `0.4.0`, and `None`.

Reply:
My recommendation is
```
DEVELOPMENT_CONTRACT_VERSION = None
```

The advantages are
* the newest deployment together with service contracts
* you can see the sources on Etherscan (current `0.3._` is not available on Etherscan).

Reply:
I'm curious what happens, so I edit it...